yt_dlp/extractor/mojevideo.py

```diff
@@ -0,0 +1,35 @@
+from .common import InfoExtractor
+
+
+class MojevideoIE(InfoExtractor):
+    _VALID_URL = r'https?://(www\.)?mojevideo\.sk/video/(?P<id>\w+)'
+
+    _TESTS = [
+        {
+            'url': 'https://www.mojevideo.sk/video/3d17c/chlapci_dobetonovali_sme_mame_hotovo.html',
+            'md5': '384a4628bd2bbd261c5206cf77c38c17',
+            'info_dict': {
+                'id': '250236',
+                'ext': 'mp4',
+                'title': 'Chlapci dobetónovali sme, máme hotovo! - Mojevideo',
+                'description': 'Celodenná práca bola za pár sekúnd fuč. Betón stiekol k susedovi, kam aj zrútil celý plot, ktorý polámal aj tuje....'
+            }
+        }
+    ]
+
+    def _real_extract(self, url):
+        display_id = self._match_id(url)
+        webpage = self._download_webpage(url, display_id)
+
+        v_id = self._search_regex(r'\bvId=(\d+)', webpage, 'video id')
+        v_exp = self._search_regex(r'\bvEx=\'(\d+)', webpage, 'expiry')
+        v_hash = self._search_regex(r'\bvHash=\[([^\]]+)', webpage, 'hash').split(",")[0].replace("'", "")
```

Comment:
Does this not work?
```suggestion
        v_hash = self._search_regex(r'\bvHash=\[\'(\w+)', webpage, 'hash')
```

Reply:
![image](https://github.com/yt-dlp/yt-dlp/assets/89145825/7544dee7-3d0d-425d-b829-97fe5df51c5f)
it doesnt